docs/pages/image-extractor.rst

```diff
@@ -4,31 +4,82 @@ Habitat Sim Image Extractor Tutorial
 .. contents::
     :class: m-block m-default
 
+In this tutorial we will learn how to use the image extraction API in Habitat Sim by working through an example use case - training a semantic segmentation model. The tutorial will be structured as follows. We will first provide an overview of how to use the image extraction API. Next we will show how to use the API to train a semantic segmentation model. Finally, we will show how to define a custom pose extraction method, which is how a user programmatically defines the camera poses for image extraction.
```

Comment:
This is great overall, but one thing lacking is a description of the "default" pose extractor. How exactly do you place cameras in a mesh? Basically, before I can decide if I want to write my own pose extraction method, I want to know what the default is.